docs/editor/intellisense.md

```diff
@@ -0,0 +1,156 @@
+---
+Order: 4
+Area: editor
+TOCTitle: IntelliSense
+ContentId: 4E9A74AA-D778-4D1C-B369-83763B3C340F
+PageTitle: IntelliSense in Visual Studio Code
+DateApproved: 8/15/2016
+MetaDescription: One of the core features of Visual Studio Code is IntelliSense.  Set breakpoints, step-in, inspect variables and more.
+---
+
+# IntelliSense
+
+IntelliSense is a general term for a variety of code editing features, including: code completion, parameter info, quick info, and list members. IntelliSense is described in a number of ways such as "code completion", "content assist", and "code hinting."
+
+![IntelliSense demo](images/intellisense/intellisense.gif)
+
+## Where is IntelliSense for my language? 
+
+IntelliSense is provided in VS Code for JavaScript, TypeScript, JSON, HTML, CSS, Less, and Sass out of the box. Other languages have word completions and can be configured to have IntelliSense by installing a language extension. 
+
+Below are the most popular language extensions in the [Marketplace](https://marketplace.visualstudio.com/vscode). Click on an extension tile below to read the description and reviews to decide which extension is best for you. 
+
+> **Tip:** For configuring and troubleshooting JavaScript IntelliSense see [here](/docs/languages/javascript#configuring-intellisense).
+
+<div class="marketplace-extensions-languages"></div>
+
+## IntelliSense Features
+
+If a language service knows possible completions, the IntelliSense suggestions will pop up as you type. If you continue typing characters, the list of members (variables, methods, etc.) is filtered to include only members containing your typed characters. Pressing `kbstyle(Tab)` or `kbstyle(Enter)` will insert the selected member. 
+
+You can trigger IntelliSense in any editor window by typing `kb(editor.action.triggerSuggest)` or by typing a trigger character (such as `.` in JavaScript). 
+
+![intellisense in package json](images/intellisense/intellisense_packagejson.gif)
+
+> **Tip:** The suggestions filtering supports CamelCase so you can type the upper case letters of a method name to limit the suggestions. For example, "cra" will quickly bring up "createApplication".
+
+> **Tip:** You can turn off IntelliSense while you type. See [Customize IntelliSense below](#_customize_intellisense).
+
+In addition, you can see **quick info** for each method (as provided by the language service). 
+
+![quick info](images/intellisense/quick_outline.png)
+
+After choosing a method you are provided with **parameter info**. 
```

Comment:
It might be worth mentioning what `any` means. Something along the lines of: 
_The language service surfaces the underlying types in the quick info and method signatures provided. In the image above, you can see several `any` types. Because JavaScript is dynamic and doesn't require or enforce types, `any` suggests that the variable can be anything._